README.md

```diff
@@ -119,6 +119,21 @@ files: {
 }
 ```
 
+Alternatively, you can specify your wn function to do the concatenation. Here's an exaple that
```

Comment:
Thanks for adding an example for the documentation. Looks great.

Sorry, only little typos here: `wn` -> `own`, `exaple` -> `example`.